python/.env.example

```diff
@@ -38,3 +38,11 @@ BEEAI_LOG_LEVEL=INFO
 
 # XAI_API_KEY=your-xai-api-key
 # XAI_CHAT_MODEL=grok-2
+
+########################
```

Comment:
this still needs to be looked into - exactly what variables are exposed & how auth should be done. Then the references here & elsewhere will be updated